pydoclint-baseline.txt

```diff
@@ -0,0 +1,233 @@
+src/braket/aws/aws_device.py
+	DOC101: Method `AwsDevice.run_batch`: Docstring contains fewer arguments than in function signature.
```

Comment:
Do we still need this?

Reply:
No, I shouldn't have copied this file over. Removing it in https://github.com/amazon-braket/autoqasm/pull/5.

Reply:
Just saw that! Thanks!